src/Reflection/BetterReflection/SourceLocator/OptimizedDirectorySourceLocatorFactory.php

```diff
@@ -42,12 +40,11 @@ public function createByDirectory(string $directory): NewOptimizedDirectorySourc
 		$files = $this->fileFinder->findFiles([$directory])->getFiles();
 		$fileHashes = [];
 		foreach ($files as $file) {
-			try {
-				$contents = FileReader::read($file);
```

Comment:
I think there are more places that do the same thing, please check ResultCacheManager. Or all the places that use FileReader. Thanks.